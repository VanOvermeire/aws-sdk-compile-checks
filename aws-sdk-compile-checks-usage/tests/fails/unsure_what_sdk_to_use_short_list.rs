use aws_sdk_compile_checks_macro::required_props;
use aws_sdk_amplify::Client;

#[required_props]
async fn do_call(client: Client) {
    client.list_artifacts()
        .send()
        .await
        .expect("Call to succeed");
}

fn main() {}
