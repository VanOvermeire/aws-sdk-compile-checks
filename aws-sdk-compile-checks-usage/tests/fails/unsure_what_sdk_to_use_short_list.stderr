error: method `list_artifacts` is used in multiple SDKs: amplify, devicefarm, sagemaker. Please add the right one(s) to the attribute, e.g. `#[required_props(sdk = amplify)]`
 --> tests/fails/unsure_what_sdk_to_use_short_list.rs:6:12
  |
6 |     client.list_artifacts()
  |            ^^^^^^^^^^^^^^
